Makefile

```diff
@@ -26,5 +26,15 @@ build-hugo:
 .PHONY: serve
 serve: build-hugo
 	@mkdir -p ./resources/_gen/images && mkdir -p ./resources/_gen/assets
-	@${CONTAINER_RUNTIME} run -t -i --sig-proxy=true --rm --mount type=bind,src=$(shell pwd),dst=/site -w /site -p 1313:1313 kiali/hugo:${HUGO_VERSION} /hugo serve --baseURL "http://localhost:1313/" --bind 0.0.0.0 --disableFastRender
+	@${CONTAINER_RUNTIME} run -t -i --sig-proxy=true --rm -v "$(shell pwd)":/site:z -w /site -p 1313:1313 kiali/hugo:${HUGO_VERSION} /hugo serve --baseURL "http://localhost:1313/" --bind 0.0.0.0 --disableFastRender
+	@rm -rf ./resources
+
+# This will ignore old versioned documentation for serving, to speed up the generation process
+serve-latest: build-hugo
+	@mkdir -p ./resources/_gen/images && mkdir -p ./resources/_gen/assets && mkdir -p ./.ignore
+	@mv  ./content/documentation/v* ./.ignore
+	# move back just latest
+	@mv  "./.ignore/$(shell basename $(shell readlink -f content/documentation/latest))" ./content/documentation
```

Comment:
Perhaps this line confused me, I haven't yet tested locally, but "latest" link doesn't point to "staging" folder, but last released version.

Reply:
what it does is copying all "v*" folders to ".ignore/", except the one that corresponds to "latest" link (which is put back to its place as a second step). The "staging" folder doesn't match "v*" so it is not moved.

Reply:
let me add a comment